src/js/page/main-controller.js

```diff
@@ -128,6 +130,39 @@ export default class MainController {
     }
   }
 
+  _onGlobalPaste(event) {
+    let val = event.clipboardData.getData('text').trim();    
+    if (!val.includes("</svg>")) {
+      this._toastsUi.show("Pasted value not an SVG", {
+        duration: 2000
+      });
+    }
+    else {
+      this._mainMenuUi.setPasteInput(val);
+      this._toastsUi.show("Paste completed", {
+        duration: 2000
+      });
```

Comment:
```suggestion
```

The toast is great for copy, but I don't think it's needed for paste. The UI already makes it clear that something happened.